### PR TITLE
feat: ✨ add sd-container - CSS style

### LINKED ISSUE #271: feat: ✨ add sd-container - CSS style
## Description
As a user of the Solid Design System, I would like to utilize `sd-container`, so that I can structure content on the page.

## API
[Figma Link](https://www.figma.com/file/VTztxQ5pWG7ARg8hCX6PfR/Solid-DS-%E2%80%93-Component-Library?type=design&node-id=1664-15968&mode=design)

## Documentation
[Figma Link](https://www.figma.com/file/ffdz9zO1CISlr8aHCZ7Bzp/Container?type=design&node-id=1001-3239&mode=design&t=Uq3OmZrR6FzL3iHN-11)

## Hand-off
[Figma Link](https://www.figma.com/file/ffdz9zO1CISlr8aHCZ7Bzp/Container?type=design&node-id=6-4&mode=design)

#### Classes
| Status | Name | Type | Default | Description |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| <ul><li>[ ] </li></ul>  | `--variant-` | `neutral-100` / `primary-100` / `primary` / `border-neutral-400` / `white` | `neutral-100` | defines the background color and border of the container |
| <ul><li>[ ] </li></ul>  | `--padding-` | `sm` / `lg` | `lg` | defines the padding of the container to be used on smaller component widths |

#### Stories (besides the `default` story)
  - [ ] `variant` (white container on primary background)
  - [ ] `default-padding` (shows the different paddings)
  - [x] `custom-padding` (shows how to override padding)

## DoR
- [ ] Item has business value
- [ ] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Changes have been pushed and reviewed
- [ ] Documentation has been created/updated
- [ ] Migration Guide has been created/updated
- [ ] Implementation works successfully on `feature` branch
- [ ] Automatic tests have passed on `feature` branch
- [ ] Accessibility has been approved on `feature` branch
- [ ] Changes have been merged (`feature` branch has been deleted)
- [ ] Automatic tests have passed on `main` branch
- [ ] Accessibility has been approved on `main` branch

### PR DESCRIPTION
**Pull request by Vahid1919**
## Description:
Added sd-container. Closes #271.

## Definition of Reviewable:
*PR notes: Irrelevant elements should be removed.*
- [x] Documentation is created/updated
- [x] Stories (features, a11y) are created/updated
- [x] relevant tickets are linked
- [x] PR is assigned to project board

### CODE REVIEW

**Comment by coraliefeil** [comment]
Approved from design side. Thanks a lot!

**Comment by yoezlem** [comment]
Is it possible to leave the default values selected?
image

File: packages/components/src/styles/container/container.declaration.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import type { Style } from '../../declaration';
+
+export default {
+  styleName: 'sd-container',
+  summary:
+    'Container lets users delimit and highlight a piece of content. The user has no interaction with it, it is merely a visual element that influences the flow of the page.',
+  status: 'stable',
+  since: '1.29.0',
```
Comment (karlbaumhauer) [inline packages/components/src/styles/container/container.declaration.ts:8]: pls adopt before merging

**Review by karlbaumhauer — APPROVED** [review]
approved with version hint before merging

**Comment by Vahid1919** [comment]
> Is it possible to leave the default values selected? image

@yoezlem It's not very straightforward since these are styles and not strictly attributes. It might take some time to try to figure out. I noticed the other style stories are similar.

**Comment by karlbaumhauer** [comment]
> > Is it possible to leave the default values selected? 
> 
> @yoezlem It's not very straightforward since these are styles and not strictly attributes. It might take some time to try to figure out. I noticed the other style stories are similar.

@Vahid1919 maybe you can open a ticket to find a way and change it in all style-components at once. I support the request from @yoezlem but dont see this blocking here.

**Comment by Vahid1919** [comment]
> @Vahid1919 maybe you can open a ticket to find a way and change it in all style-components at once. I support the request from @yoezlem but dont see this blocking here.


Created ticket here: https://github.com/solid-design-system/solid/issues/625

**Review by abudd1094 — APPROVED** [review]
Busy Vahid!

**Review by yoezlem — APPROVED** [review]

**Review by azraefendic — APPROVED** [review]

**Pull request merged**